ovh/resource_dedicated_server_install_task.go

```diff
@@ -105,17 +93,25 @@ func resourceDedicatedServerInstallTask() *schema.Resource {
 							ForceNew:    true,
 							Description: "",
 						},
-						"ssh_key_name": {
+					},
+				},
+			},
+			"user_metadata": {
+				Type:     schema.TypeList,
+				Optional: true,
+				ForceNew: true,
+				MaxItems: 10,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"key": {
 							Type:        schema.TypeString,
+							Description: "The key for the user_metadata",
 							Optional:    true,
```

Comment:
```suggestion
							Optional:    false,
```
Does it work ? Just key is not optional, value should remain optional

Reply:
c'est pour la description du schema; là c'est optionnel